### DISCUSSION TITLE
loading time has some delay?

### DISCUSSION THREAD

**Discussion by rodhash** [nvim-neo-tree/neo-tree.nvim#1410] [Q&A]
nvim-tree opens instantly to me but neo-tree seems to have a sort of small delay, anyone else noticed it? Didn't find other thread about this..

Once neo-tree is opened I don't see any difference it's only the loading time that caught my attention, so I was wondering this is normal or it could be some bad config on my end.

**Comment by arash-jafarpour** [comment]
I noticed the same thing with neo-tree didn't try nvim-tree though

**Comment by pysan3** [comment]
Yes you might notice some delay, especially due to async loading of the information of the files.

**Especially on Windows**.

If you disable git related information, you might get better performance.

FYI, performance improvement is the first thing on our TODO list and it is WIP: https://github.com/nvim-neo-tree/neo-tree.nvim/discussions/1340#discussioncomment-8659184
